src/spacel/provision/db/rds.py

```diff
@@ -0,0 +1,219 @@
+from botocore.exceptions import ClientError
+import logging
+
+from spacel.provision import clean_name, bool_param
+from spacel.provision.db.base import BaseTemplateDecorator
+
+logger = logging.getLogger('spacel.provision.rds.factory')
+
+DEFAULT_VERSIONS = {
+    'mysql': '5.7.10',
+    'postgres': '9.5.2'
```

Comment:
I didn't want to add _too_ much without testing it, and tbh I only ever tested `postgres`.

You _can_ use `mariadb` free-hand, you just need to provide `version` and `port` parameters.